### PR TITLE
Installation page: use 'main' branch, add direct artifact links

### PR DESCRIPTION
**Pull request by szarnyasg**
This PR adds direct links for the artifacts created by the nightly build using artifacts.duckdb.org.

The MacOS build currently fails so those links fail because there is no older build for `main` to fall back to (only `master`): https://github.com/duckdb/duckdb/actions?query=branch:main+event:repository_dispatch+workflow:OSX

**Status:** this is blocked on the MacOS CI build being broken. A fix is proposed at https://github.com/duckdb/duckdb/pull/8679

### CODE REVIEW

**Comment by szarnyasg** [comment]
So the links work with `wget` but not with `curl`.
See https://stackoverflow.com/questions/20986395/why-does-curl-not-work-but-wget-works
This can be fixed by using `curl -L`. I added example commands to the boxes in the Installation page. Browsers can follow the links just fine.

**Comment by szarnyasg** [comment]
This should only be merged once the OS X build is fixed, which is expected happen after https://github.com/duckdb/duckdb/pull/8674

**Comment by carlopi** [comment]
It would be nice to eventually have a complete script, like:

```
wget artifacts.duckdb.org/linux.zip
unzip linux.zip
unzip linux_cli.zip
chmod +x duckdb
./duckdb --version
```

**Comment by szarnyasg** [comment]
My biggest discovery today is that you can save one line using `jar` to unzip:
```sh
curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-linux.zip | jar x
jar xf duckdb_cli-linux-amd64.zip
chmod +x duckdb
./duckdb
```

**Comment by carlopi** [comment]
At the cost of an extra dependency / black magic feel.

**Comment by szarnyasg** [comment]
Sure - it's not something I'd put on the Installation page :).

**Review by Mause — CHANGES_REQUESTED** [review]
You'll need to update the scripts that correspond to this functionality as well

**Comment by szarnyasg** [comment]
Good point, I updated the JS.

**Comment by szarnyasg** [comment]
This PR fixes #1030 using artifacts.duckdb.org

**Comment by Mause** [comment]
> This PR fixes #1030 using artifacts.duckdb.org

Er, 1030 is this pr?

**Comment by szarnyasg** [comment]
Oh no, we have fallen into a recursion loop :)

**Comment by szarnyasg** [comment]
This PR fixes #281 using artifacts.duckdb.org

File: _includes/installation.html [hunk 1]
```diff
@@ -201,16 +201,16 @@ <h3>Usage Example</h3>
...
 
-	<div class="master r">
+	<div class="main r">
 		{% highlight r %}install.packages('duckdb', repos=c('https://duckdb.r-universe.dev', 'https://cloud.r-project.org')){% endhighlight %}
 	</div>
 
-	<div class="master java">
-		Java master builds are available in the sonatype snapshots repository, which can be accessed with the following:
+	<div class="main java">
+		Java main builds are available in the sonatype snapshots repository, which can be accessed with the following:
```
Comment (Mause) [inline _includes/installation.html:213]:
```suggestion
		Java dev builds are available in the sonatype snapshots repository, which can be accessed with the following:
```

File: _includes/installation.html [hunk 2]
```diff
@@ -234,36 +234,44 @@ <h3>Usage Example</h3>
...
-	<div class="master cli binary linux">
-		Linux Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3ALinuxRelease" target="_blank">the "LinuxRelease" CI runs</a>
-	</div>
+	<div class="main cli binary cplusplus macos">
+		MacOS binaries (universal): <a href="https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip" target="_blank">https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip</a><br/><br/>
 
-	<div class="master cli binary win">
-		Windows Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3AWindows" target="_blank">the "Windows" CI runs</a>
+		wget https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip<br/>
+		curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip -o duckdb-binaries-osx.zip
```
Comment (Mause) [inline _includes/installation.html:262]:
```suggestion
		curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-osx.zip -O
```

File: _includes/installation.html [hunk 3]
```diff
@@ -234,36 +234,44 @@ <h3>Usage Example</h3>
...
 	</div>
 
+	<div class="main cli binary cplusplus win odbc">
+		Win 64-bit: <a href="https://artifacts.duckdb.org/latest/duckdb-binaries-windows.zip" target="_blank">https://artifacts.duckdb.org/latest/duckdb-binaries-windows.zip</a><br/><br/>
+
+		wget https://artifacts.duckdb.org/latest/duckdb-binaries-windows.zip<br/>
+		curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-windows.zip -o duckdb-binaries-windows.zip
```
Comment (Mause) [inline _includes/installation.html:269]: Curl isn't really used on Windows, except as the alias inside powershell (maybe just use invoke web request directly instead?)

**Review by Mause — CHANGES_REQUESTED** [review]
I'm not sure how important providing commands is? I think it's more important to have clickable links

Either way, some minor corrections

File: _includes/installation.html [hunk 4]
```diff
@@ -234,36 +234,44 @@ <h3>Usage Example</h3>
...
 
-		<p>Linux Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3ALinuxRelease" target="_blank">the "LinuxRelease" CI runs</a></p>
+		$ wget https://artifacts.duckdb.org/latest/duckdb-binaries-linux.zip<br/>
+		$ curl -L https://artifacts.duckdb.org/latest/duckdb-binaries-linux.zip -o duckdb-binaries-linux.zip<br/><br/>
 
-		<p>Windows Build Artifacts are available from <a href="https://github.com/duckdb/duckdb/actions?query=branch%3Amaster+event%3Arepository_dispatch+workflow%3AWindows" target="_blank">the "Windows" CI runs</a></p>
-	</div>
+		Linux arm64/aarch64: <a href="https://artifacts.duckdb.org/latest/duckdb-binaries-linux-aarch64.zip" target="_blank">https://artifacts.duckdb.org/latest/duckdb-binaries-linux-aarch64.zip</a><br/><br/>
 
-	<div class="master cli binary macos">
```
Comment (hannes) [inline _includes/installation.html:254]: was this div deleted by accident?

**Comment by szarnyasg** [comment]
> I would also not add curl commands at all, just show the URL so people can click

I removed the commands

**Pull request merged**